src/core_modules/capture-core/components/DataEntries/SingleEventRegistrationEntry/DataEntryWrapper/DataEntry/DataEntry.component.js

```diff
@@ -579,16 +579,9 @@ class NewEventDataEntry extends Component<Props> {
                         {...passOnProps}
                     />
                 </div>
-                <div
-                    className={classes.savingContextContainer}
-                >
-                    <InfoIcon />
-                    <div
-                        className={classes.savingContextText}
-                    >
-                        {this.getSavingText()}
-                    </div>
-                </div>
+                <InfoIconText
+                    text={this.getSavingText()}
+                />
```

Comment:
![image](https://user-images.githubusercontent.com/4181674/102347376-9f9b3b80-3fa0-11eb-8163-ff2ba6374292.png)
updates the component we used in the event programs for uniformity across the app